uefi/src/table/runtime.rs

```diff
@@ -533,3 +571,17 @@ pub struct VariableStorageInfo {
     /// Maximum size of an individual variable of the specified type.
     pub maximum_variable_size: u64,
 }
+
+/// Information about UEFI variable storage space returned by
+/// [`RuntimeServices::query_capsule_capabilities`].
+#[derive(Clone, Copy, Debug, Default, Eq, PartialEq)]
+pub struct CapsuleInfo {
+    /// The maximum size in bytes that UpdateCapsule()
```

Comment:
nit: Change `UpdateCapsule()` to doclink to the `update_capsule` function (ditto for the `query_capsule_capabilities` docstring)